pysages/methods/metad.py

```diff
@@ -287,3 +288,24 @@ def sum_of_gaussians(xi, heights, centers, sigmas, periods):
     """
     delta_x = wrap(xi - centers, periods)
     return gaussian(heights, sigmas, delta_x).sum()
+
+
+@dispatch
+def analyze(result: Result[Metadynamics]):
+    """
+    Computes the free energy from the result of a `Metadynamics` run.
+    """
+    method = result.method
+    state = result.states
+
+    P = get_periods(method.cvs)
+    heights = state.heights
+    centers = state.centers
+    sigmas = state.sigmas
+
+    @jit
```

Comment:
It's nice to compile this with jit, what we are not expecting this to be performance critical, right?

Reply:
I think `metapotential` evaluation depends on the number of bins used in the CV for computing free energy. So, it can be expensive for 2D or higher dimensions. However, not sure how much performance gain we obtain using `jit` here.

Reply:
Yeah, I think it is fine. @pabloferz is there a drawback that we should consider here for users?

Reply:
I don't think it will hurt even if this not performance critical, and I don't think this will be problematic for the end-user, they should be able to plot or write to file `DeviceArray`s in the same way they do with `nd.array`s.